.github/workflows/scripts/dist-diff.mjs

```diff
@@ -0,0 +1,49 @@
+import { readFile } from 'node:fs/promises'
+
+import { comment, githubActionArtifactsUrl } from './github.mjs'
+
+const DIST_DIFFS = [
```

Comment:
Can we pass this in from the GitHub Action YAML?

Then we can keep everything in **comments.mjs** with maybe:

1. Comment as `comment()`
2. Comment single diff as `commentDiff()`
3. Comment multiple diffs as `commentDiffs()`

Preserves the change history from @domoscargin too